src/server.py

```diff
@@ -5,13 +5,31 @@
 from fastapi import FastAPI, HTTPException
 from fastapi.staticfiles import StaticFiles
 from deck import Deck
+from fastapi import Query
+from card import Card
+import secrets
+import string
 
 SUITS = ["C", "D", "H", "S"]
 RANKS = ["2", "3", "4", "5", "6", "7", "8", "9", "0", "J", "Q", "K", "A"]
+allDecks = {}
+
+
+# fully made by chatGPT, secrets is supposed to be cryptographically secure so hopefully that 
```

Comment:
Yeah, this should work. The secrets module does indeed give a cryptographically secure random number generator.